_aggregations/metric/geobounds.md

```diff
@@ -7,11 +7,22 @@ redirect_from:
   - /query-dsl/aggregations/metric/geobounds/
 ---
 
-## Geobounds aggregations
+# Geobounds aggregations
 
-The `geo_bounds` metric is a multi-value metric aggregation that calculates the [geographic bounding box](https://docs.ogc.org/is/12-063r5/12-063r5.html#30) containing all values of a given `geo_point` or `geo_shape` field. The bounding box is returned as the upper-left and lower-right vertices of the rectangle in terms of latitude and longitude.
+The `geo_bounds` metric is a multi-value metric that calculates the [geographic bounding box](https://docs.ogc.org/is/12-063r5/12-063r5.html#30) encompassing all of a set of `geo_point` or `geo_shape` objects. The bounding box is returned as the upper-left and lower-right vertices of the rectangle given as a decimal-encoded latitude-longitude (lat-lon) pair.
 
-The following example returns the `geo_bounds` metrics for the `geoip.location` field:
+## Parameters
+
+The `geo_bounds` aggregation takes the following parameters: 
+
+| Parameter        | Required/Optional | Data type      | Description |
+| :--              | :--               | :--            | :--         |
+| `field`          | Required          | String         | Name of the field containing the geopoints or geoshape for which the geobound is computed. |
```

Comment:
```suggestion
| `field`          | Required          | String         | The name of the field containing the geopoints or geoshapes for which the geobounds are computed. |
```

Reply:
It's not clear to me whether you can provide more than one `geoshape` to the aggregation?

Reply:
@rishabhmaurya Is it possible to provide more than one geoshape to the aggregation?

Reply:
that's the name of the field and its accepting name of the field on which one is aggregating

Reply:
Thanks! @dwelsch-esi For every document, there is one field that contains either a geopoint or a geoshape. So I think using plural for both geopoint and geoshape is fine because it conveys an aggregation on a set of geopoints and geoshapes for all documents.